test/DocumentFormat.OpenXml.Tests/SaveAndCloneTests.cs

```diff
@@ -33,7 +33,7 @@ public void CanCloneDocument()
                 {
                     var body = clone.MainDocumentPart.Document.Body;
                     body.InsertBefore(new Paragraph(new Run(new Text("Hello World"))), body.FirstChild);
-                    clone.SaveAs(file.Path).Close();
+                    clone.SaveAs(file.Path).Dispose();
```

Comment:
Please refactor things such that we never call dispose directly

Reply:
yup